x-pack/legacy/plugins/siem/public/store/network/reducer.ts

```diff
@@ -301,27 +309,27 @@ export const networkReducer = reducerWithInitialState(initialNetworkState)
       flowTarget,
     },
   }))
-  .case(updateTlsLimit, (state, { limit }) => ({
+  .case(updateTlsLimit, (state, { limit, networkType }) => ({
     ...state,
-    [NetworkType.details]: {
-      ...state[NetworkType.details],
+    [networkType]: {
```

Comment:
I got type errors and had to do the specific check like you see in `updateTopNFlowLimit`. have you run a type check here, trying to run it locally and its blowing up